frontend/src/components/regions/AddCardReviewModal/index.js

```diff
@@ -44,13 +44,18 @@ const AddReviewModalUnconnected = ({
     saveReview({ status, comments, cardId });
   };
 
+  const closeHelpPopUp = (e) => {
+    document.getElementById("popUpInfo").remove();
```

Comment:
I think we should use React mechanisms here, don't interact directly with the dom.  Are we managing the opening and closing of this modal using state or redux?  Please use the mechanisms in place.